packages/integration-tests/bin/cli.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const pckg = require('../package.json');
+const testRunner = require('../index');
+const program = require('commander');
+
+program.version(pckg.version);
+
+/**
+ * Verify that the given param is not null. Write out an error if null.
+ *
+ * @param {Object} paramConfig - param name and value {name: value:}
+ * @returns {boolean} true if param is not null
+ */
+function verifyRequiredParameter(paramConfig) {
+  if (paramConfig.value === null) {
+    console.log(`Error: ${paramConfig.name} is a required parameter.`);
+    return false;
+  }
+
+  return true;
+}
+
+/**
+ * Verify required parameters are present
+ *
+ * @param {list<Object>} requiredParams - params in the form {name: 'x' value: 'y'}
+ * @returns {boolean} - true if all params are not null
+ */
+function verifyWorkflowParameters(requiredParams) {
+  return requiredParams.map((param) => verifyRequiredParameter(param)).includes(false) === false;
```

Comment:
Instead of `requiredParams.map((param) => verifyRequiredParameter(param))`, just `requiredParams.map(verifyRequiredParameter)`.